src/main/scala/com/ironcorelabs/DeviceAddResult.scala

```diff
@@ -0,0 +1,41 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+import java.{util => ju}
+
+/**
+ * Account's device context. Needed to initialize the SDK with a set of device keys.
+ *
+ * @param userId ID of a user
+ * @param segmentId ID of a segment
+ * @param devicePrivateKey asymmetric private key
+ * @param signingPrivateKey signing private key specific to a device
+ * @param deviceId ID of the device that was added
+ * @param name name of the device that was added
+ * @param created date that the device was created
+ * @param lastUpdated date that the device was last updated
+ */
+case class DeviceAddResult(
+  userId: UserId,
+  segmentId: Long,
+  devicePrivateKey: PrivateKey,
+  signingPrivateKey: DeviceSigningPrivateKey,
+  deviceId: DeviceId,
+  name: Option[DeviceName],
+  created: ju.Date,
+  lastUpdated: ju.Date
+)
```

Comment:
Should this case class have a `toDeviceContext` to represent the easy conversion?

Reply:
I was thinking a constructor for `DeviceContext` that takes a `DeviceAddResult` would be cleaner. And I thought I had done that but obviously I'm crazy.
Which way would you prefer? (or both?)

Reply:
Both sounds great.

Reply:
Added both